src/ecs/systems/client/HandleIncomingMessages.hpp

```diff
@@ -358,10 +364,24 @@ namespace ecs::systems
         std::cerr << "Error: Client couldn't kill unknown entity with netId '" << msgId << "'." << std::endl;
     }
 
+    static void playerHealthHandle(World &world, network::Message &msg)
+    {
+        // size_t msgId = (unsigned char)msg[1] << 8U | (unsigned char)msg[2];
+
+        auto &healths = world.registry.getComponents<component::Health>();
+        for (size_t i = 0; i < healths.size(); i++) {
+            if (healths[i]) {
+                healths[i].value().health = msg[1];
+            }
+        }
```

Comment:
Add a verification to ensure that the netId is the same as the client's to avoid updating everyone's health bar (ask @EmileEttel for a How-To)